src/Symfony/Bridge/Doctrine/Security/User/EntityUserProvider.php

```diff
@@ -44,7 +44,7 @@ public function __construct(ManagerRegistry $registry, string $classOrAlias, str
     /**
      * {@inheritdoc}
      */
-    public function loadUserByUsername($username)
+    public function loadUserByUsername(string $username)
```

Comment:
💡 These changes to `doctrine-bridge` 5 break compatibility with `security-core` 4.4, thus the bump in composer.json. Can be reverted and re-applied later, if you don't want to bump yet.